### DISCUSSION TITLE
Background color of NPM payment receipt email may confuse users

### DISCUSSION THREAD

**Discussion by sigveio** [npm/feedback#508] [General]
This is how an email receipt for a successful subscription payment to NPM arrives in my email inbox:

![npm-receipt-small](https://user-images.githubusercontent.com/6737410/127547049-33ba527a-1ae7-48d9-aff8-4b7b4a9de808.png)

I feel the red background color is a bit unfortunate.

Every time I see it, my brain automatically goes **“oh, oh, has the payment failed!?”.**

Most people subconsciously associate green with good, yellow/orange with potential problem, red with STOP… danger... problem. Especially in the context of computers, and even more so with billing/payments.

- I would suggest changing to a more neutral background color.
- Or differentiate and use green for successful payments, red for failed.

This would avoid unnecessary confusion.

**Comment by MylesBorins** [comment]
Hey @sigveio . I've opened a ticket with engineering to get the color updated. Thank you for reporting

**Reply by ethomson** [reply]
Hi @sigveio - this isn't what we're seeing in Gmail (web client) or Outlook.  Can you tell us what mail user agent you're using?

**Reply by ljharb** [reply]
I see this exact screenshot’s red color in gmail (the black boxes are just redaction)

**Reply by ethomson** [reply]
Odd.  This is what we have configured (and what I see on my receipts).  We'll have to dig in a little deeper to understand this.

![image](https://user-images.githubusercontent.com/1130014/131889172-072a9b9e-6433-407f-8144-22e503dd7438.png)

**Reply by ljharb** [reply]
I'm in Safari; are you perhaps using Chrome? CSS in emails is highly picky about browsers, way more than webpages, unfortunately :-/

**Reply by ethomson** [reply]
No, I'm in Safari and checked it with Chrome and Edge as well.  You'll note that the content is very different as well.  I suspect that we're getting legitimately different receipt templates based on some variable - for example, maybe you're on an older billing "plan" and the email receipt template is configured with the plan.  Not sure yet but it's surprisingly a bit less trivial than I would have expected.  Regardless, we'll sort this out.

**Reply by ljharb** [reply]
I’m definitely on a legacy plan, called “the free plan” that ironically isn’t free :-)
